src/CallbackInvoker.php

```diff
@@ -28,6 +28,6 @@ public function __construct(ContainerInterface $container)
             new TypeHintResolver,
             new TypeHintContainerResolver($container),
             new NumericArrayResolver,
-        ]));
+        ]), $container);
```

Comment:
This looked like a bug to me, at least my implementation didn't work without this. 

Without this, the CallbackInvoker doesn't send `$container` to it's parent, so when a `call()` was made, it wouldn't actually do a lookup in the container...